core/src/test/java/org/elasticsearch/index/query/SimpleQueryStringBuilderTest.java

```diff
@@ -177,15 +178,6 @@ public void testNullQueryTextGeneratesException() {
         assertThat(exception, notNullValue());
     }
 
-    @Test
-    public void testHandlingDefaults() throws IOException {
```

Comment:
I think what it was supposed to test is that `toQuery()` uses reasonable defaults if some parameters (like analyzer) are not set in the builder. However, it seems that this is already covers in the randomized builder setup, so if we just want to check that `toQuery` produces something in these cases, this should be covered.